content/client-server-api/_index.md

```diff
@@ -739,7 +739,7 @@ submit:
 ```
 
 Alternatively reply using a 3PID bound to the user's account on the
-homeserver using the `/account/3pid`\_ API rather than giving the `user`
+homeserver using the `/account/3pid` API rather than giving the `user`
```

Comment:
They used to be links in RST. Let's make them continue to be!

```suggestion
homeserver using the [`/account/3pid`](#get_matrixclientv3account3pid) API rather than giving the `user`
```